### PR TITLE
LPS-168680: Remove input permission from UI after blog creation

### PR DESCRIPTION
**Pull request by kristofloneck**
Brian's review: https://github.com/brianchandotcom/liferay-portal/pull/128529

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by kristofloneck** [comment]
Changes after brian review.

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1467,6 +1467,22 @@ public BlogsEntry updateStatus(
...
+					addEntryResources(
+						entry, serviceContext.isAddGroupPermissions(),
+						serviceContext.isAddGuestPermissions());
+				}
+				else {
+					addEntryResources(
+						entry, serviceContext.getModelPermissions());
+				}
+			}
+
```
Comment (adolfopa) [inline modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java:1485]: @kristofloneck The code for both conditions is identical. Can you merge both conditions?
```
		if ((oldStatus == WorkflowConstants.STATUS_DRAFT) ||
                     GetterUtil.getBoolean(
				serviceContext.getAttribute("addEntryResources"))) {

				if (serviceContext.isAddGroupPermissions() ||
					serviceContext.isAddGuestPermissions()) {

					addEntryResources(
						entry, serviceContext.isAddGroupPermissions(),
						serviceContext.isAddGuestPermissions());
				}
				else {
					addEntryResources(
						entry, serviceContext.getModelPermissions());
				}
			}
```

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java (hunk 1 above)
Comment (kristofloneck) [reply modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java]: fixed

**Comment by kristofloneck** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by Hanlf** [comment]
ci:forward

**Comment by Hanlf** [comment]
The failures are not related to this PR, sent it to BChan: https://github.com/brianchandotcom/liferay-portal/pull/128965

**Pull request closed without merging**